### ISSUE TITLE
Wrong pre-processing steps

### ISSUE THREAD

**Issue by KaeLiuChenyu** [Chang-Li-HFUT/ACRNN#5]
If I'm not mistaken, the data preparation process is to first window the data, then shuffle it, and finally split the training and test sets.

This is a typical data leakage, where the test set and training set have a serious crossover and you can achieve close to 100% accuracy with whatever network you use, which after all is basically the same as training with the test set.

The correct approach is to shuffle first and then window the test and training sets separately. Hope you can provide the right results, a near-100% accuracy rate is ridiculous.
